calculator_api_server/main.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import FastAPI, Request
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
+from schemas import CommonResponseModel
+
+app = FastAPI()
+
+
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(_: Request, exc: RequestValidationError):
+    message = exc.errors()[0]["msg"]
+    return JSONResponse(content={"error": message}, status_code=400)
+
+
+@app.get('/add', response_model=CommonResponseModel)
+async def get_addition_result(number1: float, number2: float):
+    return JSONResponse(content={"data": {"result": number1 + number2}})
+
+
+@app.get('/subtract', response_model=CommonResponseModel)
+async def get_subtraction_result(number1: float, number2: float):
+    return JSONResponse(content={"data": {"result": number1 - number2}})
```

Comment:
pydantic model을 리턴하면 알아서 해준답니다

```suggestion
    return CommonResponseModel(data=CalculationResult(result=number1 - number2)
```

Reply:
https://github.com/eunnbi/zero-to-hero/pull/1/commits/c854b7cfdbc1045e00a339e25cba2609bd8bbf6c